src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.classes`. In this case, instead of styling `.stars__star`, consider adding a class to each `div` inside `.stars` and style that class.